js/directives/day-block.js

```diff
@@ -15,7 +15,7 @@
 			'</div>'+
 
 			'<div ng-style="controller.monthStyleOverride()" ' +
-			'class="month">{{controller.date | luifMoment: \'MMM\' | limitTo : 3}}'+
+			'class="month">{{controller.date | luifMoment: \'MMM\' | fancyMonth}}'+
```

Comment:
I suppose that, as the day-block should be used also inside the figgo-mobile app @harrich and @thomaf could give us some opinion about it.